src/Umbraco.Web.UI.Client/src/packages/documents/document-types/entity-actions/create/default/default-document-type-create-option-action.ts

```diff
@@ -0,0 +1,23 @@
+import { UMB_CREATE_DOCUMENT_TYPE_WORKSPACE_PATH_PATTERN } from '../../../paths.js';
+import type { UmbDocumentTypeEntityTypeUnion } from '../../../entity.js';
+import { UmbEntityCreateOptionActionBase } from '@umbraco-cms/backoffice/entity-create-option-action';
+import type { MetaEntityCreateOptionAction } from '@umbraco-cms/backoffice/entity-create-option-action';
+
+export class UmbDefaultDocumentTypeCreateOptionAction extends UmbEntityCreateOptionActionBase<MetaEntityCreateOptionAction> {
+	override async execute() {
```

Comment:
Since we are generating a HREF/URL, then we should not use an exeucte method, but instead use the getHref method. so the actual URL is exposed to the user and the user can use such to open in a new tab etc. More transparent and accessible. I have corrected this.